GENESIS.md

```diff
@@ -101,19 +100,61 @@ contract tokenFallback {
 //       it to do.
 contract TokenBurner {
     struct Claim {
-        uint256 amount;
-        bytes32 pubkey;
+        uint256[] amount;
+        bytes32[] pubkey; // if people burn tokens multiple times, we can either 
+                          //1. store every AE-pubkey or 
+                          //2. store only those which havn't been stored before, which would require a seperate mapping + array to iterate over, so I suggest we go with option 1.
+    }
+    
+    
+    // Store the hashes of Aadmins' msg.data used for counting up the AE delivery batch
+    mapping (address => bytes32) private multiSigHashes;
+    // Keep track of token burn batches - use this number for filtering in the emitted Burn event.
+    uint16 public AEdeliveryBatchCounter = 0;
+    
+    // The admins which may count up the AE delivery batch count
```

Comment:
Do we need a multisig here? A rogue admin could only ever increment the batch number and not wreak any havoc that way—unless we consider the possibility of overflowing the counter.

I think I'm a bit more concerned with one private key going missing and then not being able to increment batch number.

What do you think?

Reply:
I think so too, let's skip multisig because no real harm can be done, but having access when we need it is essential.